runelite-client/src/main/java/net/runelite/client/config/ConfigDescriptor.java

```diff
@@ -24,26 +24,22 @@
  */
 package net.runelite.client.config;
 
+import lombok.Getter;
 import java.util.Collection;
 
 public class ConfigDescriptor
 {
+	@Getter
```

Comment:
If all fields have `@Getter`, you can annotate the class with it instead for brevity.